datajoint/fetch.py

```diff
@@ -199,15 +153,11 @@ def __getitem__(self, item):
         :return: tuple with an entry for each element of item
 
         Examples:
-
         >>> a, b = relation['a', 'b']
         >>> a, b, key = relation['a', 'b', datajoint.key]
-        >>> results = relation['a':'z']    # return attributes a-z as a tuple
```

Comment:
I agree that positional insert contradicts the above principle.  I was hesitant to add it.  But the coding and execution efficiency improvement is much greater for the positional insert than for fetch slicing.

Slicing of the form `["string":"string":3]` is unfamiliar and therefore not expressive to new users.